chip-var.cwl

```diff
@@ -137,10 +149,16 @@ steps:
         source: snpsift_countOpName
       - id: snpsift_prevalOpName
         source: snpsift_prevalOpName
+      - id: opOncoKbMafName
+        source: opOncoKbMafName
+      - id: oncoKbApiToken
+        source: oncoKbApiToken
+      - id: oncoKbAnnotateHotspots
+        source: oncoKbAnnotateHotspots
     out:
       - id: cosmicCount_annotatedOutput
       - id: annotatedOutput_prevalence
-      - id: vcf2maf_maf
+      - id: opOncoKbMaf
```

Comment:
@karthigayini  can we make sure we have output of each step

Reply:
sure. Will make the changes